darts/models/forecasting/torch_forecasting_model.py

```diff
@@ -1429,6 +1419,105 @@ def _load_from_checkpoint(self, file_path, **kwargs):
         pl_module_cls = getattr(sys.modules[self._module_path], self._module_name)
         return pl_module_cls.load_from_checkpoint(file_path, **kwargs)
 
+    def load_weights_from_checkpoint(
+        self,
+        model_name: str = None,
+        work_dir: str = None,
+        file_name: str = None,
+        best: bool = True,
+        strict: bool = True,
+        **kwargs,
+    ):
+        """
+        Load only the weights from automatically saved checkpoints under '{work_dir}/darts_logs/{model_name}/
+        checkpoints/'. This method is used for models that were created with ``save_checkpoints=True`` and
+        that need to be re-trained or fine-tuned with different optimizer or learning rate scheduler.
```

Comment:
they don't necessarily need to be retrained , but can also just be used for prediction

Reply:
True but for this kind of use (prediction only, without changing optimizer or learning rate scheduler), wouldn't load_from_checkpoint be more appropriate?

I added the sentence, "However, it can also be used to load weights for inference."